docs/api/proposed_endpoints.md

```diff
@@ -507,3 +528,320 @@ Status: 200 OK
 }
 ```
 
+## Send notification to speaker
+
+```
+POST /api/proposal/:id/notify (requires authentication)
+```
+
+**Response**
+Status: 200 OK
+```json
+{
+    "message": "Notification of proposal accepted will be sent to the speaker"
+}
+```
+
+# Swags
+
+**Parameters**
+
+Name              | Data Type | Required | Default Value  | Discription
+------------------|-----------|----------|----------------|--------------------
+id                | UUID      | false    | _uuid_         | Unique ID for the swag.
+name              | text      | true     | ''             | Name of the swag.
+description       | text      | false    | ''             | Description of the swag.
+image             | text      | false    | ''             | URL of the image of the swag.
+created_at        | datetime  | false    | _datetime_     | Time of creation.
+modified_at       | datetime  | false    | _datetime_     | Time of modification.
+
+__Note__:
+- _uuid_: randomly generated UUID.
+- _datetime_: date and time when swag is created.
+
+```
+POST /api/swag (requires authentication)
+```
+
+**Request**
+```json
+{
+    "name": "Tshirt",
```

Comment:
this should be all lower case. There should be a mapping for FE on what to send here.

Internally this would be a choice field. The mapping can be provided in a meta-data API.

Reply:
Providing internal choice for such a broad key like `name` might not be possible because `name` can be anything like `tshirt`, `cap`, `thumb drive`, `mug`, `notebook`, `chargers`, etc. What we can do is to create a key `category` with possible general choices like `wearables`, `devices`, `kit`, `bag` and `other` (while leaving the key `name` as a common `text_field`). This way it would be more flexible.